create/how_to/aws.md

```diff
@@ -0,0 +1,181 @@
+
+# Deploy a MeiliSearch Instance on AWS
+
+Using our **MeiliSearch [AMI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html)**, MeiliSearch can be deployed on AWS in just a few minutes.
+The following `how-to guide` will walk you through every step to deploy MeiliSearch in an AWS EC2 instance.
+
+**Table of Contents**:
+
+[[toc]]
+
+## Part 1: Deploy an out-of-the-box MeiliSearch instance
+
+### 1. Launch an instance from the AWS console
+
+After logging into your [AWS Console](https://aws.amazon.com/console), navigate to the **Compute** service. Then go to **EC2**, and finally open your **Instances** console.
+
+![Launch an instance from AWS console](/aws/01.launch-instances.png)
+
+In the top-right corner, click on the **Launch instances** button to start the process of configuring your MeiliSearch instance.
+
+### 2. Select 'MeiliSearch' AMI from 'Community AMIs'
+
+You will now select which AMI or system Image to use to run your instance. Type **"meilisearch"** in the searchbar and select the **Community AMIs** tab on the left sidebar.
+
+![Choose MeiliSearch AMI from Community AMIs](/aws/02.select-ami.png)
+
+Click on **Select** (right side of the screen) to confirm your choice.
+
+### 3. Size and specs
+
+Select the specifications of the server you want MeiliSearch to run on.
+
+![Size and specs](/aws/03.size-and-specs.png)
+
+> We recommend prioritizing memory allocation for better MeiliSearch performance.
+> Note that the **free tier** is sufficient for tests or prototypes, but not recommended for large datasets.
+
+Once you've made your choice, click on **Next: Configure instance details** to continue.
+
+### 4. Instance details
+
+Here you can specify details of your MeiliSearch instance. Since **this section is not required to run MeiliSearch**, we won't cover it in this guide.
+
+![Instance details](/aws/04.instance-details.png)
+
+Simply click on **Next: Add Storage** to keep going.
+
+### 5. Storage
+
+Choose the storage **device** and **size** for your MeiliSearch instance.
+
+![Storage](/aws/05.storage.png)
+
+The amount of storage space required can vary drastically depending on the data you plan to index. In this example, we will use 25 GiB, which is more than enough for a small to medium dataset.
```

Comment:
small to medium dataset means nothing, can we be clearer

Reply:
It's a little arbitrary, but I would say that
Small dataset: less than 10,000 documents
Medium dataset: less than 1 million documents
Large dataset: 1 million documents and above
Does that sound right to you?

Reply:
But of course it's *all* arbitrary.

I'm sure you could easily pass 25gb with a dataset that has only 10,000 documents if those documents each had 1000+ fields of 1000 or more characters. There's no good way to escape the lack of clarity here, so I think maybe just change it to:

```suggestion
The amount of storage space required can vary drastically depending on the data you plan to index. In this example, we will use 25 GiB, which is more than enough for most small datasets.
```